ios/engine/KMEI/KeymanEngine/Classes/Keyboard/KeymanWebViewController.swift

```diff
@@ -131,6 +131,14 @@ class KeymanWebViewController: UIViewController {
     webView!.navigationDelegate = self
     webView!.scrollView.isScrollEnabled = false
 
+    // Disable WKWebView default layout-constraint manipulations. We ensure
+    // safe-area boundaries are respected via InputView / InputViewController
+    // constraints.
+    //
+    // Fixes #10859.
```

Comment:
If you want a complete, fully-technical explanation of how this helps to fix things, you may want to cross-reference  behavior documented here: https://github.com/keymanapp/keyman/issues/6595#issuecomment-1128575563